specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -2582,6 +2582,249 @@
+          },
+          {
+            "$ref": "#/parameters/triggerName"
+          },
+          {
+            "$ref": "#/parameters/rerunTriggerName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Trigger has been stopped successfully."
+          },
+          "default": {
+            "description": "An error response received from the Azure Data Factory service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/triggers/{triggerName}/rerunTriggers/{rerunTriggerName}/cancel": {
+      "post": {
+        "tags": [
+          "rerunTriggers"
+        ],
+        "operationId": "RerunTriggers_Cancel",
+        "x-ms-examples": {
+          "Triggers_Cancel": {
+            "$ref": "./examples/RerunTriggers_Cancel.json"
+          }
+        },
+        "description": "Cancels a trigger.",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/factoryName"
+          },
+          {
+            "$ref": "#/parameters/triggerName"
+          },
+          {
+            "$ref": "#/parameters/rerunTriggerName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Trigger has been canceled successfully."
+          },
+          "default": {
+            "description": "An error response received from the Azure Data Factory service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/triggers/{triggerName}/rerunTriggers": {
+      "get": {
+        "tags": [
+          "rerunTriggers"
+        ],
+        "operationId": "RerunTriggers_ListByTrigger",
+        "x-ms-examples": {
+          "RerunTriggers_List": {
+            "$ref": "./examples/RerunTriggers_ListByTrigger.json"
+          }
+        },
+        "description": "Lists rerun triggers by an original trigger name.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/factoryName"
+          },
+          {
+            "$ref": "#/parameters/triggerName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
```

Comment:
nit: Please stick to camelCase convention for the parameter name

Reply:
This is legacy issue. api-version is defined under parameters for a long time, and all the other APIs refer this parameter as  api-version. If we update api-version name, then all APIs using it need to be updated too. Do you think it's still necessary to update it?

Reply:
We can skip this